packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -1385,3 +1386,16 @@ export namespace DataTransfer {
         return dataTransfer;
     }
 }
+
+export function toLogLevel(logLevel: MonacoLogLevel): theia.LogLevel {
+    switch (logLevel) {
+        case MonacoLogLevel.Trace: return types.LogLevel.Trace;
```

Comment:
One more argument for using `OutputChannelImpl` as a base class: we wouldn't need this conversion.